-- Remove the -fsanitize-ignore-for-ubsan-feature flag, which is not supported
-- by the Clang version used for this FreeBSD port and causes an unknown-argument
-- error.
--- build/config/sanitizers/sanitizers.gni.orig	2026-09-24 10:57:02 UTC
+++ build/config/sanitizers/sanitizers.gni
@@ -547,9 +547,6 @@ template("ubsan_hardening") {
         # Prevents `__has_feature(undefined_behavior_sanitizer)`
         # from evaluating true. Configs defined here are intended to
         # be usable even in release builds, i.e. as widely as possible.
-        # It's important not to have full-on UBSan workarounds activate
-        # just because we built support for a specific sanitizer.
-        "-fsanitize-ignore-for-ubsan-feature=${invoker.sanitizer}",
       ]
       if (defined(invoker.cflags)) {
         cflags += invoker.cflags
